### PR TITLE
[DAT-582] add spark_and_python image

### PR DESCRIPTION
**Pull request by jamedge**
Image pubnative/spark:python3.6.5_java1.8.131_hadoop2.7.3_mesos1.3.0

### CODE REVIEW

File: spark/spark_and_python/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+RUN curl -sL --retry 3 --insecure \
+    --header "Cookie: oraclelicense=accept-securebackup-cookie;" \
+    "https://download.oracle.com/otn-pub/java/jdk/${JAVA_MAJOR_VERSION}u${JAVA_UPDATE_VERSION}-b${JAVA_BUILD_NUMBER}/d54c1d3a095b4ff2b6607d096fa80163/server-jre-${JAVA_MAJOR_VERSION}u${JAVA_UPDATE_VERSION}-linux-x64.tar.gz" \
+    | gunzip \
+    | tar x -C /usr/ \
+    && ln -s $JAVA_HOME /usr/java \
+    && rm -rf $JAVA_HOME/man
+
+# AWS
+RUN mkdir -p /opt/app/java_libs && cd /opt/app/java_libs
```
[the remark this answers is not in the document]
Comment (jamedge) [reply spark/spark_and_python/Dockerfile]: 👍

File: spark/spark_and_python/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
+FROM python:3.6.5
```
[the remark this answers is not in the document]
Comment (jamedge) [reply spark/spark_and_python/Dockerfile]: 👍

File: spark/README.md [hunk 3]
```diff
@@ -34,3 +34,11 @@ docker run --net=host pubnative/spark:2.0.2 bin/spark-submit \
...
 
 Run task from the other image
+
+# Spark with Python
+
+This image Dockerfile can be found in `python` directory. It's an image containing spark, mesos and python.
+
+## Build
+
+For building and pushing the image please look at `python/Makefile`.
```
Comment (miry) [inline spark/README.md]: Image `pubnative/spark:python3.6.5_java1.8.131_hadoop2.7.3_mesos1.3.0`

**Comment by miry** [comment]
@jamedge I see you started use Makefile. But you did not reuse ARGS :)

**Comment by jamedge** [comment]
@miry args are for java version, but as I saw the string `d54c1d3a095b4ff2b6607d096fa80163` from the java download url is also being changed (sometimes) when new version is released, so I wouldn't make bigger usage of those args :)

**Pull request merged**